### PR TITLE
Dashboard

### PR DESCRIPTION
**Pull request by ksridar**
Basic dashboard implementation and rerouted from App

### CODE REVIEW

File: src/client/components/Navbar/Navbar.jsx [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+  borderRadius: theme.shape.borderRadius,
+  backgroundColor: alpha(theme.palette.common.white, 0.15),
+  '&:hover': {
+    backgroundColor: alpha(theme.palette.common.white, 0.25),
+  },
+  marginLeft: 0,
+  width: '100%',
+  [theme.breakpoints.up('sm')]: {
+    marginLeft: theme.spacing(1),
+    width: 'auto',
```
Comment (Taqdeer) [inline src/client/components/Navbar/Navbar.jsx:32]: Just out of curiosity, why are we styling MUI components in jsx and not css?

**Review by Taqdeer — APPROVED** [review]
I think there are some conflicts with the master.

File: src/App.js [hunk 2]
```diff
@@ -1,24 +1,21 @@
 import logo from './logo.svg';
 import './App.css';
+import {
+  BrowserRouter as Router, Switch, Route, Routes, Redirect
+} from 'react-router-dom';
+
+import Dashboard from "/Users/krishsridar/Desktop/CPSC_455/Project/dogfish/src/client/components/Dashboard/Dashboard.jsx"
```
Comment (Taqdeer) [inline src/App.js:7]: I think we need a relative path here.

**Comment by srujanr40** [comment]
Merge to project_2, not main. Let's merge project_2 into main after our demo

**Pull request closed without merging**